Description: Removes explicit checking of CXX in configure.in
 This patch removes explicit checking of CXX that indeed avoids the
 --enable-cxx user option.
Author: Daniel Rus Morales <danirus@tol-project.org>

--- dmalloc-5.5.1.orig/configure.ac	2007-03-25 21:16:44.000000000 +0200
+++ dmalloc-5.5.1/configure.ac	2007-05-02 10:00:28.000000000 +0200
@@ -55,12 +55,6 @@
 AC_PROG_CC
 AC_PROG_CXX
 
-# see if we actually have a CXX program
-if test "$ac_cv_prog_CXX" = "" -o ! -x "$ac_cv_prog_CXX"; then
-    AC_MSG_WARN(could not find C++ compiler $ac_cv_prog_CXX)
-    enable_cxx=no
-fi
-
 # hopefully we have a stdc c-compiler
 if test "$ac_cv_prog_cc_stdc" = "no" ; then
 	AC_MSG_WARN()
